sdk/storage/inc/common/memory_stream.hpp

```diff
@@ -32,7 +30,7 @@ namespace Azure { namespace Storage {
     std::size_t m_offset = 0;
   };
 
-  inline std::unique_ptr<MemoryStream> CreateMemoryStream(const uint8_t* data, std::size_t length)
+  inline std::unique_ptr<MemoryStream> CreateMemoryStream(const uint8_t* data, int64_t length)
   {
     return std::make_unique<MemoryStream>(data, length);
```

Comment:
Can we remove this MemoryStream ?   Use only the one from Core now?

Reply:
Yes, I see you've changed the implementation of `Azure::Core::MemoryStream`, it doesn't make a copy now.

Reply:
done